openedx/features/philu_utils/sites.py

```diff
@@ -0,0 +1,23 @@
+"""
+All utilities for Django sites
+"""
+from logging import getLogger
+
+from django.contrib.sites.models import Site
+
+log = getLogger(__name__)
+
+
+def get_site(site_id):
+    """
+    Get site object from site id
+    Args:
+        site_id (int): the id of site
+
+    Returns:
+        site: The site object otherwise None
```

Comment:
I feel in this whole docstring we are stating the obvious, me be we can change it to one liner.